.checkstyle/checkstyle.xml

```diff
@@ -95,7 +95,9 @@
         <module name="ParameterNumber"/>
         <module name="ClassDataAbstractionCoupling"/>
         <module name="BooleanExpressionComplexity"/>
-        <module name="ClassFanOutComplexity"/>
+        <module name="ClassFanOutComplexity">
+        <property name="max" value="22"/>
+        </module>
```

Comment:
I would suppress the warning where it happens instead of setting the "right" max value here.

Reply:
Done